drivers/jdbc-driver/src/main/kotlin/com/squareup/sqldelight/sqlite/driver/JdbcDriver.kt

```diff
@@ -102,12 +102,15 @@ abstract class JdbcDriver : SqlDriver, ConnectionManager {
     binders: (SqlPreparedStatement.() -> Unit)?
   ) {
     val (connection, onClose) = connectionAndClose()
-    connection.prepareStatement(sql).use { jdbcStatement ->
-      SqliteJdbcPreparedStatement(jdbcStatement)
-        .apply { if (binders != null) this.binders() }
-        .execute()
+    try {
+      connection.prepareStatement(sql).use { jdbcStatement ->
+        SqliteJdbcPreparedStatement(jdbcStatement)
+          .apply { if (binders != null) this.binders() }
+          .execute()
+      }
+    } finally {
```

Comment:
@AlecStrong should this be happening in `finally` or in `catch`?

Reply:
finally